docs/source/kinds_of_types.rst

```diff
@@ -356,6 +356,43 @@ check against ``None`` in the if condition.
 
     ``--strict-optional`` is experimental and still has known issues.
 
+.. _noreturn:
+
+The NoReturn type
+*****************
+
+Mypy also provides an experimental support for functions that supposed to
```

Comment:
> Also, the support is not considered experimental.

But why then ``NoReturn`` is in ``mypy_extensions``? Maybe it should be moved to ``typing``?